src/sentry/static/sentry/app/components/externalIssues/abstractExternalIssueForm.tsx

```diff
@@ -59,7 +64,7 @@ export default class AbstractExternalIssueForm<
     });
   };
 
-  getConfigName = (): string => {
+  getConfigName = (): 'createIssueConfig' | 'linkIssueConfig' => {
```

Comment:
@mgaeta it might be a good idea to move this type into the `app/types` file. This seems like something that will get reused often.

Reply:
This is the only usage so far in JS but I'll move it if it is used more.

Reply:
I'm OK leaving a simple union of two types like this where it's used